src/styles/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+
+  }
+
+  &__reviews-count {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    color: $text-main-color;
+    font-weight: 700;
+  }
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    margin-top: 16px;
+    background-color: $main-accent;
+    border-radius: 4px;
+    border: 1px solid transparent;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-transform: uppercase;
+    color: #fff;
+
+    &:hover {
+      background-color: #fff;
+      border: 1px solid $main-accent;
```

Comment:
And on hover, you need to change the border-color only instead of border property